Development-Guide/CAN-Filter-Guide/README.adoc

```diff
@@ -0,0 +1,61 @@
+= CAN Message Filter Guide
+
+== Introduction
+This is a basic guide on how to make a CAN filter.
+There are two components to a CAN filter: an ID and a Mask.
+The ID is the unique identifier for the address of the message(s) being looked for, and the Mask determines which bits of the ID are important to sift through. 
+
+IMPORTANT: A Mask must be able to adapt to future additions to a CAN network. 
+It must be made *as tight as possible for longevity*.
+
+There is no set order of steps to follow, but to get started, determine the ID, then determine the Mask.
+A very useful tip is to convert the hex format into binary format, determine the ID and Mask, and then convert those back into hex.
+
+
+== Determine the ID
+The ID will be the unique identifier for the address of the message(s) being received.
+
+. The ID should be very specific and contain the important bits that will be compared to the incoming message.
+For example, if one is looking for a message from a system that sends out messages with address 0b0000_1111(0x0F) then the ID will be 0b0000_1111(0x0F) because there is only one type of address that is wanted.
+
+. The greater the variety of addresses wanted, the more flexible the ID will be to accommodate the variety of incoming messages.
+For example, if one is looking for a message from a system that sends out messages with addresses 0b0001_0001, 0b0001_0010, and 0b0001_0011 (0x11, 0x12, and 0x13) then the ID will be 0b0001_0000 (0x10) because the last two bits do not matter. 
```

Comment:
I'm a bit iffy on this, so would someone put 0s in only when the bit doesn't matter?
For example if I just had `0b001_001` and `0b001_0010`, would the id still be `0b001_0000`?